packages/orchestrator-ui-components/src/components/WFOPageTemplate/WFOSidebar/WFOSidebar.tsx

```diff
@@ -139,6 +140,16 @@ export const WFOSidebar: FC = () => {
                             },
                             href: PATH_SETTINGS,
                         },
+                        {
```

Comment:
Remove the test form page? also remove the PATH_FORMS constant

Reply:
Removed